.vscode/extensions.json

```diff
@@ -0,0 +1,7 @@
+{
+	// See http://go.microsoft.com/fwlink/?LinkId=827846
+	// for the documentation about the extensions.json format
```

Comment:
Comments are not usually permitted in JSON files, even if VSCode is OK with them, it's probably best to leave it out.

Reply:
Fixed!